pages/blog/whats-new-in-graphql-cli-4.1.0.mdx

```diff
@@ -0,0 +1,74 @@
+import withArticle from '../../ui/blog/article';
+
+export const meta = {
+  title: "What's new in GraphQL CLI 4.1",
+  author: 'ephelan',
+  tags: [
+    'graphql',
+    'node',
+    'codegen',
+    'graphql-cli',
+    'graphql-config',
+    'graphback',
+  ],
+  date: '2020-10-12',
+  description:
+    'GraphQL CLI - experience the modern way to develop full-stack GraphQL applications.',
+  image:
+    'https://user-images.githubusercontent.com/20847995/67651234-85bf1500-f916-11e9-90e5-cb3bd0e6a338.png',
+};
+
+export default withArticle({ ...meta });
+
+[GraphQL CLI](http://graphql-cli.com) is your one-stop shop for developing full-stack GraphQL applications in Node.js. Since the release of 4.0.0, we have added a number of enhancements which will further elevate your GraphQL development experience.
```

Comment:
I would add here more information, maybe things like

"With GraphQL CLI you could create a new GraphQL app in seconds! Just declare your GraphQL Schema and blablaalbalbal
and give more examples of why it's cool